core/src/main/java/org/mule/runtime/core/internal/processor/strategy/WorkQueueStreamProcessingStrategyFactory.java

```diff
@@ -133,4 +200,46 @@ public void stop() throws MuleException {
 
   }
 
+  private static final class EventWrapper {
+
+    CoreEvent wrappedEvent;
+
+    public EventWrapper(CoreEvent event) {
+      this.wrappedEvent = event;
+      ((BaseEventContext) (wrappedEvent.getContext())).getRootContext().onTerminated((e, t) -> {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove useless curly braces around statement ([squid:S1602](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS1602))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7708&issues=AWkQk2wOotIXnmsUMK6U&open=AWkQk2wOotIXnmsUMK6U)

[ISSUE_KEY]: AWkQk2wOotIXnmsUMK6U